scripts/find_pulsar_in_obs.py

```diff
@@ -210,10 +210,18 @@ def grab_source_alog(source_type='Pulsar', pulsar_list=None, max_dm=1000., inclu
                 for entry in columns:
                     if entry not in ['', ' ', '\t']:
                         temp.append(entry.replace('--',''))
+                #Removing bad formating for the database
+                ra = temp[4]
```

Comment:
We really shouldn't have variables named 'temp' every variable serves some purpose and can be described better than that